### PR TITLE
Display the source on which the 'fuzzer' failed

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
Ah this has to reworked because of `set -e `.

**Comment by chriseth** [comment]
Perhaps like that?

File: test/cmdlineTests.sh [hunk 1]
```diff
@@ -77,10 +77,17 @@ TMPDIR=$(mktemp -d)
     REPO_ROOT=$(pwd) # make it absolute
     cd "$TMPDIR"
     "$REPO_ROOT"/scripts/isolate_tests.py "$REPO_ROOT"/test/
+    set +e
     for f in *.sol
     do
         "$REPO_ROOT"/build/test/solfuzzer --quiet < "$f"
+        if [ $? -ne 0 ]; then
+            echo "Fuzzer failed on:"
+            echo $f
```
Comment (axic) [inline test/cmdlineTests.sh]: This needs to be `cat $f`.

**Comment by axic** [comment]
@chriseth updated this, can you double check?

File: test/cmdlineTests.sh [hunk 2]
```diff
@@ -79,7 +79,14 @@ TMPDIR=$(mktemp -d)
     "$REPO_ROOT"/scripts/isolate_tests.py "$REPO_ROOT"/test/
     for f in *.sol
     do
+        set +e
         "$REPO_ROOT"/build/test/solfuzzer --quiet < "$f"
+        if [ $? -ne 0 ]; then
+            echo "Fuzzer failed on:"
+            cat $f
+            exit 1
```
Comment (axic) [inline test/cmdlineTests.sh]: Must delete the temporary directory here.

**Pull request merged**